_people/KarinaLiles.md

```diff
@@ -0,0 +1,12 @@
+---
+first_name: Karina
+last_name: Liles
+title: Assistant Professor of Mathematics and Computer Science
+school: Clafin University
+website: https://www.claflin.edu/academics-research/faculty-research/meet-our-faculty/dr.-karina-liles
+image: https://www.claflin.edu/images/default-source/faculty-profiles/dr-lyles-profile.jpg
+---
+Prof. Liles directs the [Social Technologies and Robotics (STAR) Lab](https://www.drkliles.com/STARlab). Her work centers on human-robot interaction and educational robotics, specifically for improving the quality of life for at need populations. 
+* J. M. Beer, M. Boren and K. R. Liles, "[Robot assisted music therapy a case study with children diagnosed with autism](https://doi.org/10.1109/HRI.2016.7451785)," 2016 11th ACM/IEEE International Conference on Human-Robot Interaction (HRI), 2016, pp. 419-420.
```

Comment:
We don't have a consistent citation format between people, but at least within one bio lets pick a format and stick to it. The other two have the date before the title and the title not in quotes.